examples/jsm/controls/OrbitControls.js

```diff
@@ -292,6 +287,23 @@ class OrbitControls extends EventDispatcher {
 
 		}();
 
+		this.connect = function (domElement) {
+
+			if ( domElement === document ) console.error( 'THREE.OrbitControls: "document" should not be used as the target "domElement". Please use "renderer.domElement" instead.' );
+
+			this.domElement = domElement
+			this.domElement.style.touchAction = 'none'; // disable touch scroll
+
+			scope.domElement.addEventListener( 'contextmenu', onContextMenu );
```

Comment:
`this`, not `scope`.

Reply:
Right, idk why there is a `scope` variable at all? Would be nice to keep symmetry with the `dispose` method and use the same there

Reply:
actually it needs to be `scope`, `this` refers to the function's context not the class

Reply:
> function's context not the class

But they are the same.

Reply:
Or you can just use an anonymous function.

Reply:
Yeah you're right, although I still think `connect`/`dispose` should follow the same pattern. I can update `dispose` if that is something you want

Reply:
Maybe then it would be better to create a function `disconnect`? And call it from `dispose` (possibly with a warning thrown).

Reply:
Yep, `disconnect` sounds more natural with `connect`. But isn't `dispose` used everywhere to perform cleanup for other resources (geometries etc). So there could be valuable to keep that consistency. Maybe `attach` is a better name for `connect`?

Reply:
Then I just suggest to use `this.dispose = this.disconnect`.

Reply:
(And that would be useful for other controls, which may require to do in `dispose` something other than just `disconnect`)